src/app/facility_manager/facilityprojectsoverview.component.html

```diff
@@ -538,31 +546,78 @@ <h4 class="modal-title" *ngIf="emailStatus === 2"> Error </h4>
 </div><!-- /.modal -->
 
 
-<div bsModal #descriptionModal="bs-modal" class="modal fade" tabindex="-1" role="dialog"
-     aria-labelledby="myModalLabel" aria-hidden="true">
-  <div class="modal-dialog modal-info" role="document">
-    <div class="modal-content">
-      <div class="modal-header">
-        <h4 class="modal-title">Description</h4>
-        <button type="button" class="close" style="cursor: pointer"
-                (click)="descriptionModal.hide()"
-                aria-label="Close">
-
-          <span aria-hidden="true">&times;</span>
-        </button>
-      </div>
-
-      <div class="modal-body">
-        <p>{{ selectedProject?.Description }}</p>
-      </div>
-      <div class="modal-footer">
-        <button type="button" class="btn btn-primary"
-                (click)="selectedProject=undefined;descriptionModal.hide();">
-          Close
-        </button>
-      </div>
-    </div><!-- /.modal-content -->
-  </div><!-- /.modal-dialog -->
+<!--<div bsModal #descriptionModal="bs-modal" class="modal fade" tabindex="-1" role="dialog"-->
```

Comment:
why is the description modal commented out?

Reply:
Not sure, but couldn't find any usage of it anywhere, is it still used? Forgot to undo this, sorry!